### PR TITLE
Adjust Mobile View for Account Overview and Course List

### LINKED ISSUE #582: Mobile View of Account Overview as Admin Does Not Fit Screen
image

- fix add button to only show icon
- remove matriculation number 
- (adjust the filter buttons to fill the complete width)

### PR DESCRIPTION
**Pull request by theovier**
# Description

- Fixes #582

## Reason for this PR
- the mobile view of the account overview was not really useable

## Changes in this PR
- make changes to mobile ( + partly to desktop) version to increase usability

Old Mobile:
image

New Mobile:
image

Old Desktop:
image

New Desktop:
1/2
image

2/2
image

## Type of change
- Refactoring

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Manually tested all use cases this could alter

**Test Configuration**:
- OS: Windows Mac Linux
- Browser: Firefox Chrome Safari Chromium

- Frontend:
  - [x] Development build
- Backend:
  - [x] No backend needed to test this

# Checklist:

- [x] I have performed a self-review of my own code
- [x] My changes generate no new linting warnings or console warnings
- [x] I have updated the CHANGELOG.md to include any changes made in this PR (add WIP to the top, if there is none already)

### CODE REVIEW

**Comment by theovier** [comment]
I just realised that I should probably adjust the course overview too with the same changes as otherwise, the UX is different. Will do this in the coming days before merging.

**Comment by bastihav** [comment]
> I just realised that I should probably adjust the course overview too with the same changes as otherwise, the UX is different. Will do this in the coming days before merging.

Sounds good. I like the changes.

**Comment by theovier** [comment]
> 
> Sounds good. I like the changes.
> We should merge this relatively soon though, as we can add profile picture thumbnails very soon (already implemented in lagom, but not deployed)

Acknowledged. I will try to have everything asap, so we can merge it.

**Comment by theovier** [comment]
I decided to align everything but the name on the right-hand side, so the name can take up as much space as it needs. This prevents the situation where names were cropped but not the full width was used.

image

File: src/components/course/list/common/CourseTypeFilter.vue [hunk 1]
```diff
@@ -39,9 +37,21 @@
...
+                switch (type) {
+                    case CourseType.PG: {
+                        return "PG";
+                    }
+                    default: {
+                        return type;
+                    }
+                }
+            }
+
```
Comment (bastihav) [inline src/components/course/list/common/CourseTypeFilter.vue:50]: Do you always need a short form? As I explained yesterday, CourseTypes will soon be a configuration by the backend, so we would not know the values on compile time. We could maybe propose pairs of type + short form

**Review by bastihav — APPROVED** [review]

File: src/components/course/list/common/CourseTypeFilter.vue (hunk 1 above)
Comment (theovier) [reply src/components/course/list/common/CourseTypeFilter.vue:50]: I think that would be nice to always have a short-form for the mobile UI.

**Pull request merged**